src/color/readableColor.js

```diff
@@ -26,13 +35,30 @@ import getLuminance from './getLuminance'
  *   color: "#fff";
  *   color: "#ff8";
  *   color: "#001";
+ *   color: "#000";
  * }
  */
 
+const defaultLightReturnColor = '#000'
+const defaultDarkReturnColor = '#fff'
+
 export default function readableColor(
   color: string,
-  lightReturnColor: string = '#000',
-  darkReturnColor: string = '#fff',
+  lightReturnColor: string = defaultLightReturnColor,
+  darkReturnColor: string = defaultDarkReturnColor,
+  strict: boolean = false,
```

Comment:
Not something you introduced, but let's notate these as optional `?` for Flow.

Reply:
I work with TypeScript, and thought this looked wrong - but lazily assumed there was something flow-related going on. Sorry, updated!